services/order.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import Order, Ticket
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: datetime | None = None) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+
+        order.save()

```

Comment:
i may be wrong, but isn't it better to place the `save` statement in the `if` block so as not to make a query when not needed?